six/modules/c++/six.sicd/include/six/sicd/CropUtils.h

```diff
@@ -29,11 +29,32 @@
 #include <types/RowCol.h>
 #include <scene/Types.h>
 #include <six/NITFReadControl.h>
+#include <six/sicd/ComplexData.h>
 
 namespace six
 {
 namespace sicd
 {
+/*!
+ * Projects a slant plane pixel into ground plane LLA coordinates
```

Comment:
It's just lat, lon (not lat, lon, alt, even though we have that internally initially), so change comment that says LLA to something like lat/lon